### PR TITLE
Check shebang in bash scripts

### PR DESCRIPTION
**Pull request by wtatarski**
Related to discussion: https://github.com/SymbiFlow/yosys-symbiflow-plugins/pull/115#discussion_r645724460

### CODE REVIEW

**Comment by wtatarski** [comment]
@mithro

File: checks/checks.py [hunk 1]
```diff
@@ -120,6 +120,23 @@ def report_error(error_message):
     print(f'::error::{error_message}')
     return [error_message]
```
Comment (mithro) [inline checks/checks.py:122]: Two new lines between top level functions.

**Review by mithro — COMMENTED** [review]
Please add tests.

File: checks/checks.py (hunk 1 above)
Comment (wtatarski) [reply checks/checks.py:122]: Fixed

**Review by mithro — APPROVED** [review]
LGTM.

**Pull request merged**